sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/WatchFeature.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration;
+
+import com.azure.data.appconfiguration.models.ConfigurationSetting;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class WatchFeature {
+    /**
+     * Runs the sample algorithm and demonstrates how to read configuration setting revision history.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(String[] args) {
+        // The connection string value can be obtained by going to your App Configuration instance in the Azure portal
+        // and navigating to "Access Keys" page under the "Settings" section.
+        String connectionString = "endpoint={endpoint_value};id={id_value};secret={secret_value}";
```

Comment:
Should we gear samples towards using connection strings or AAD?

Reply:
since other samples already using connection string. We can follow the same pattern. ConnectionString is easier to use when people trying to run samples.